test/functional/feature_csv_activation.py

```diff
@@ -381,8 +396,9 @@ def run_test(self):
         self.log.info("BIP 112 tests")
         self.log.info("Test version 1 txs")
 
-        # -1 OP_CSV tx should fail
+        # -1 OP_CSV tx and (empty stack) OP_CSV tx should fail
         self.send_blocks([self.create_test_block([bip112tx_special_v1])], success=False)
+        self.send_blocks([self.create_test_block([bip112tx_emptystack_v1])], success=False)
```

Comment:
Let's check the reject reason

Reply:
Done in commit https://github.com/bitcoin/bitcoin/pull/17921/commits/3330bcec758e83e2e76f8d4f1cca1c4c3d543a83.